tools/checkCopyright.js

```diff
@@ -34,7 +34,7 @@ var CopyrightVerifier = (function () {
             return defer.resolve(foundFiles);
         });
         finder.on("patherror", function (err, strPath) {
-            defer.reject("Error for Path " + strPath + " " + err);
+            defer.reject("Error for Path " + strPath + " " + err + "\nStackTrace: " + err.stack);
```

Comment:
The docs don't really say, but I'm choosing to believe that err will never be null. Why would you call a patherror event if you don't have an error?